examples/8b_benchmark_read_parallel.cpp

```diff
@@ -744,9 +744,9 @@ main( int argc, char *argv[] )
     MPI_Comm_size( MPI_COMM_WORLD, &input.m_MPISize );
     MPI_Comm_rank( MPI_COMM_WORLD, &input.m_MPIRank );
 
-    if (argc < 2) {
+    if (argc < 3) {
       if (input.m_MPIRank == 0)
-          std::cout<<"Usage: "<<argv[0]<<" input_file_prefix"<<std::endl;
+          std::cout<<"Usage: "<<argv[0]<<" input_file_prefix pattern"<<std::endl;
```

Comment:
Please document the new syntax in the manual here: `docs/source/usage/benchmarks.rst` :)

Reply:
It is already documented there, only the code did not print any message if no pattern was provided.

Reply:
I think I am looking for a line similar to
```
$ cat --help
Usage: cat [OPTION]... [FILE]...
Concatenate FILE(s) to standard output.
```
For instance:
```
Usage: 8a_benchmark_write_parallel <input-file> [pattern]
``
in case that `<input-file>` is mandatory and `[pattern]` is optional

Reply:
I see, this is the content, just written in a free format instead of using the unix usual conventions:
https://github.com/openPMD/openPMD-api/blob/dev/docs/source/usage/benchmarks.rst#reading

Already there, does not need to be improved in the current PR then (but we should do that eventually @guj )

Reply:
The input-fie is mandatory and pattern is optional. When pattern is not specified,  the old default behavior is to run all patterns.   It is later changed  to not doing anything. 

It is ok if you both like  forcing the pattern to be specified.

Reply:
Oh we want to keep it optional - I was just saying that we need to improve the doc page to show a standard UNIX-like, easy to read usage in it.